install.sh

```diff
@@ -7,23 +7,23 @@ apt-get clean
 apt-get update
 
 echo "================ Installing gradle ================="
-wget -nv https://services.gradle.org/distributions/gradle-4.8.1-all.zip
-unzip -qq gradle-4.8.1-all.zip -d /usr/local && rm -f gradle-4.8.1-all.zip
-ln -fs /usr/local/gradle-4.8.1/bin/gradle /usr/bin
-echo 'export PATH=$PATH:/usr/local/gradle-4.8.1/bin' >> /etc/drydock/.env
+wget -nv https://services.gradle.org/distributions/gradle-4.9-all.zip
```

Comment:
use env here. no point repeating the same thing over and over again

Reply:
ok